scripts/package.ps1

```diff
@@ -41,4 +45,8 @@ docker build `
     -t $IMAGE `
     -f Dockerfile .
 
+if ($LASTEXITCODE -ne 0) {
```

Comment:
Required so that Docker build failures actually cause a failure on PR workflows; before, these errors were squashed and never returned.